lte/gateway/python/magma/enodebd/enodeb_status.py

```diff
@@ -333,10 +341,21 @@ def get_operational_states(
     states = []
     configured_serial_ids = []
     enb_status_by_serial = get_all_enb_status(enb_acs_manager)
+
+    # Get S1 connected eNBs
+    enb_s1_state_map = get_all_enb_state()
+
     for serial_id in enb_status_by_serial:
         enb_status_dict = enb_status_by_serial[serial_id]._asdict()
```

Comment:
Nit for here and elsewhere: [avoid Hungarian notation](https://docs.quantifiedcode.com/python-anti-patterns/readability/putting_type_information_in_a_variable_name.html). Name the variable something semantically meaningful. Then use typing information to handle the rest

E.g. here I like just `enb_statuses`